### ISSUE TITLE
Installer throws ERR_REQUIRE_ESM when making a clean install

### ISSUE THREAD

**Issue by kluplau** [remix-run/remix#3044]
### What version of Remix are you using?

latest

### Steps to Reproduce

npx create-remix@latest --template remix-run/blues-stack NAME

### Expected Behavior

Would install a project in a folder called NAME.

### Actual Behavior

It throws an error

```
➜  projects npx create-remix@latest --template remix-run/blues-stack PROJECT_NAME

Need to install the following packages:
  create-remix@latest
Ok to proceed? (y) y
npm WARN deprecated source-map-url@0.4.1: See https://github.com/lydell/source-map-url#deprecated
npm WARN deprecated urix@0.1.0: Please see https://github.com/lydell/urix#deprecated
npm WARN deprecated resolve-url@0.2.1: https://github.com/lydell/resolve-url#deprecated
npm WARN deprecated rollup-plugin-inject@3.0.2: This package has been deprecated and is no longer maintained. Please use @rollup/plugin-inject.
npm WARN deprecated source-map-resolve@0.5.3: See https://github.com/lydell/source-map-resolve#deprecated
/Users/my-username/.npm/_npx/5164864a48bff686/node_modules/@remix-run/dev/cli/create.js:26
var sortPackageJSON = require('sort-package-json');
                      ^

Error [ERR_REQUIRE_ESM]: require() of ES Module /Users/my-username/.npm/_npx/5164864a48bff686/node_modules/sort-package-json/index.js from /Users/my-username/.npm/_npx/5164864a48bff686/node_modules/@remix-run/dev/cli/create.js not supported.
Instead change the require of index.js in /Users/my-username/.npm/_npx/5164864a48bff686/node_modules/@remix-run/dev/cli/create.js to a dynamic import() which is available in all CommonJS modules.
    at Object.<anonymous> (/Users/my-username/.npm/_npx/5164864a48bff686/node_modules/@remix-run/dev/cli/create.js:26:23)
    at Object.<anonymous> (/Users/my-username/.npm/_npx/5164864a48bff686/node_modules/@remix-run/dev/cli/commands.js:29:16)
    at Object.<anonymous> (/Users/my-username/.npm/_npx/5164864a48bff686/node_modules/@remix-run/dev/cli/run.js:24:16)
    at Object.<anonymous> (/Users/my-username/.npm/_npx/5164864a48bff686/node_modules/@remix-run/dev/cli/index.js:15:11)
    at Object.<anonymous> (/Users/my-username/.npm/_npx/5164864a48bff686/node_modules/@remix-run/dev/index.js:15:13)
    at Object.<anonymous> (/Users/my-username/.npm/_npx/5164864a48bff686/node_modules/create-remix/cli.js:14:11) {
  code: 'ERR_REQUIRE_ESM'
}
```

**Comment by pollingj** [comment]
Just to confirm, I'm having the same issue as well.

**Comment by kluplau** [comment]
I tried to reinstall node, reboot machine, and install an earlier version of the installer, but with no luck.

**Comment by pollingj** [comment]
It's because the latest version of sort-package-json is using ESM

**Comment by chungweileong94** [comment]
> It's because the latest version of sort-package-json is using ESM

Yeah, the latest package is causing the ESM error
https://github.com/keithamus/sort-package-json/compare/v1.55.0...v1.56.0?short_path=b335630#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5

**Comment by byhemechi** [comment]
A very quick fix is to replace the version of sort-package-json in the npx cache with version 1.55.0

e.g. for the error
```
Error [ERR_REQUIRE_ESM]: require() of ES Module /Users/zsty/.npm/_npx/82f938b29eb562d9/node_modules/sort-package-json/index.js
```

the problem would be solved with 

```bash
cd /Users/zsty/.npm/_npx/82f938b29eb562d9/
npm i sort-package-json@1.55.0
```

**Comment by pSnehanshu** [comment]
Facing the same issue here.

**Comment by thisislawatts** [comment]
Yep, looks like this change from `sort-package-json` _should_ have been a breaking change but unfortunately semantic release can be a little hard to work with here. 😓 
https://github.com/keithamus/sort-package-json/pull/256#issuecomment-1087653594

**Comment by ByronBos** [comment]
Easiest fix in the short term to is to override the version in your package.json, got my project back up and running.

`"overrides": {
    "sort-package-json": "1.55.0"
  }`

**Comment by rabilrbl** [comment]
> Easiest fix in the short term to is to override the version in your package.json, got my project back up and running.
> 
> `"overrides": { "sort-package-json": "1.55.0" }`

I am unable to run `create-remix` for init template.

**Comment by ByronBos** [comment]
> 

Ah yes apologies, my comment is for existing projects you might be trying to work with.
I would guess the fix @byhemechi mentioned would be the only immediate fix for that.

You could potentially clone one of the examples from https://github.com/remix-run/remix/tree/main/examples and then add the override as another way to get going.

**Comment by garth** [comment]
https://github.com/keithamus/sort-package-json/issues/258

**Comment by kukuricapolna** [comment]
> A very quick fix is to replace the version of `sort-package-json` in the npx cache with version 1.55.0
> 
> e.g. for the error
> 
> ```
> Error [ERR_REQUIRE_ESM]: require() of ES Module /Users/zsty/.npm/_npx/82f938b29eb562d9/node_modules/sort-package-json/index.js
> ```
> 
> the problem would be solved with
> 
> ```shell
> cd /Users/zsty/.npm/_npx/82f938b29eb562d9/
> npm i sort-package-json@1.55.0
> ```

thanks this works

**Comment by wdcarlos** [comment]
Here instead of ```overrides``` i got it working by adding ```"resolutions": { "sort-package-json": "1.55.0" }``` to my package.json

**Comment by keithamus** [comment]
This should be fixed in v1.56.1
